src/agent/Makefile

```diff
@@ -27,6 +27,14 @@ COMMIT_MSG = $(if $(COMMIT),$(COMMIT),unknown)
 # Exported to allow cargo to see it
 export VERSION_COMMIT := $(if $(COMMIT),$(VERSION)-$(COMMIT),$(VERSION))
 
+##VAR SECCOMP=yes|no define if agent enables seccomp feature
+SECCOMP := no
+
+# Enable seccomp feature of rust build
```

Comment:
Suggest: explicitly declare `SECCOMP_FEATURES` like `SECCOMP_FEATURES :=`

Reply:
Thank you for the suggestion.
I fixed it and changed the name of `SECCOMP_FEATURES` to `SECCOMP_FEATURE`.